customers/schema/types.py

```diff
@@ -145,13 +148,25 @@ def resolve_winter_storage_applications(self, info, **kwargs):
     def __resolve_reference(self, info, **kwargs):
         profile = get_node_from_global_id(info, self.id, only_type=ProfileNode)
         return return_node_if_user_has_permissions(
-            profile, info.context.user, CustomerProfile, BerthApplication, BerthLease
+            profile,
+            info.context.user,
+            CustomerProfile,
+            BerthApplication,
+            BerthLease,
+            Order,
+            BerthSwitchOffer,
```

Comment:
We have to add winter leases, since they can also be queried through the node.

Reply:
and I think boats as well

Reply:
Added